presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/fragments/youtubefragments/diffUtil/VideoDiffUtil.kt

```diff
@@ -1,4 +1,4 @@
-package com.safetyheads.akademiaandroida.youtube.diffUtil
+package com.safetyheads.akademiaandroida.presentation.ui.fragments.youtubefragments.diffUtil
```

Comment:
why files in diffUtil aren't in the same package as rest of youtube related classes?